components/Conversation/V3Conversation.tsx

```
@@ -0,0 +1,335 @@
+    ({ item }) => {
+      const message = messages?.byId[item];
+      const fromMe =
+        message?.senderAddress.toLowerCase() === currentAccount.toLowerCase();
+      if (!message) return null;
+
+      const contentType = getMessageContentType(message?.contentTypeId);
+      switch (contentType) {
+        // case "groupUpdated":
+        //   return <ChatGroupUpdatedMessage message={message} />;
+        case "text":
+          let content = "";
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)